l1-contracts/src/core/libraries/MerkleLib.sol

```diff
@@ -50,4 +50,33 @@ library MerkleLib {
       revert Errors.MerkleLib__InvalidRoot(_expectedRoot, subtreeRoot);
     }
   }
+
+  /*
+  * @notice Calculates a tree height from the amount of elements in the tree
+  * @param _size - The amount of elements in the tree
+  */
+  function calculateTreeHeightFromSize(uint256 _size) internal pure returns (uint256) {
```

Comment:
See comment from earlier. You should be able to derive it from the content commitment values.